bindings/python/sample_network_auditing.py

```diff
@@ -7,7 +7,14 @@
 from unicorn.x86_const import *
 import struct
 import uuid
-import random
+
+# Python 2/3 Compat without installing six
+DEAD_PYTHON = False
+import sys
+
+if sys.version_info[0] < 3:
+    DEAD_PYTHON = True
+    print("Python 2.x is dead. Start thinking about migrating to 3.x. https://wiki.python.org/moin/Python2orPython3")
```

Comment:
I've spent more time reading this PR than any computer will ever waste in `iteritems()` when running this sample. Just switch to `items()` for Python 2.x and drop the DEAD_PYTHON checks.